### PR TITLE
[FIX] ExpressionChangedAfterItHasBeenCheckedError when using lu-select

### PR DESCRIPTION
**Pull request by GuillaumeNury**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://458-pr-lucca-front-LuccaSA.surge.sh

File: packages/ng/libraries/core/src/lib/api/select/input/api-select-input.component.ts [hunk 1]
```diff
@@ -124,7 +123,8 @@ implements ControlValueAccessor, ILuInputWithPicker<T>, AfterViewInit {
 		$event.stopPropagation();
 		$event.preventDefault();
 	}
-	ngAfterViewInit() {
+	ngAfterContentInit() {
+		this._isContentInitilized = true;
```
Comment (lucienbertin) [inline packages/ng/libraries/core/src/lib/api/select/input/api-select-input.component.ts]: typo here: `this._isContentInitialized`, with a `a`

File: packages/ng/libraries/core/src/lib/select/input/select-input.component.ts [hunk 2]
```diff
@@ -131,7 +130,8 @@ implements ControlValueAccessor, ILuInputWithPicker<T>, AfterViewInit {
 	}
 
 
-	ngAfterViewInit() {
+	ngAfterContentInit() {
+		this._isContentInitilized = true;
```
Comment (lucienbertin) [inline packages/ng/libraries/core/src/lib/select/input/select-input.component.ts]: same

File: packages/ng/libraries/core/src/lib/select/input/select-input.model.ts [hunk 3]
```diff
@@ -27,6 +27,8 @@ implements ControlValueAccessor, ILuInputWithPicker<T>, ILuInput<T> {
 		);
 	}
 	protected _placeholder: string;
+	protected _isContentInitilized = false;
```
Comment (lucienbertin) [inline packages/ng/libraries/core/src/lib/select/input/select-input.model.ts]: also

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://458-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**